Add kilted as active distribution to welcome page.

rkent opened the pull request:
I just noticed the home page does not list kilted, so this adds it.

rkent committed:
Add kilted as active distribution to welcome page.

Signed-off-by: R Kent James <kent@caspia.com>
Generated-by: Portions of this commit may include code completion from github.copilot version 1.372.0 or later

christophebedard commented on _layouts/search_packages.html:
```diff
@@ -75,6 +75,9 @@ <h2>Active Distributions</h2>
         <a href="https://docs.ros.org/en/jazzy/Releases/Release-Jazzy-Jalisco.html">ROS 2 Jazzy</a>
       </p>
+      <p>
+        <a href="https://docs.ros.org/en/jazzy/Releases/Release-Kilted-Kaiju.html">ROS 2 Kilted</a>
```
Just to match the other links:

```suggestion
        <a href="https://docs.ros.org/en/kilted/Releases/Release-Kilted-Kaiju.html">ROS 2 Kilted</a>
```

rkent replied:
Nice catch, I guess I was a little too quick to toss this off!

Committed in my branch.

rkent committed:
Fix typo while adding kilted.

Signed-off-by: R Kent James <kent@caspia.com>
Generated-by: Portions of this commit may include code completion from github.copilot version 1.372.0 or later

christophebedard reviewed (approved):
LGTM.

(I feel like we should flip the order and go from newest to oldest, but we can do that later.)
